lib/naver/searchad/api/core/base_service.rb

```diff
@@ -1,3 +1,11 @@
+require 'addressable/uri'
+require 'addressable/template'
+require 'httpclient'
+require 'naver/searchad/api/options'
```

Comment:
ok will think about this later again, but since this is out of review scope :) not changing here :)